Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -1893,6 +1893,13 @@ GUniquePtr<GstStructure> GStreamerMediaEndpoint::preprocessStats(const GRefPtr<G
         default:
             break;
         };
+
+        auto timestamp = gstStructureGet<double>(structure.get(), "timestamp"_s);
+        if (LIKELY(timestamp)) {
+            auto newTimestamp = StatsTimestampConverter::singleton().convertFromMonotonicTime(Seconds::fromMilliseconds(*timestamp));
+            gst_structure_set(structure.get(), "timestamp", G_TYPE_DOUBLE, newTimestamp.microseconds(), nullptr);
```

Comment:
?
```suggestion
            gst_structure_set(structure.get(), "timestamp", G_TYPE_DOUBLE, newTimestamp.milliseconds(), nullptr);
```

Reply:
I'm a bit confused, aren't the timestamps in milliseconds from webrtcbin's `get-stats`?

Reply:
Yes they are indeed. We convert to microseconds here to be consistent with libwebrtc, which iiuc internally uses microseconds too, hence the JSON logs are timestamped using the same semantics for both backends.

Reply:
Thanks for the clarification!